pkg/cloudprovider/vsphere/loadbalancer/README.md

```diff
@@ -21,6 +21,10 @@ required (S, M, L or XL).
 For every service port a dedicated virtual server is managed which is
 connected to this load balancer service.
 
+## Minimum Requirements
+
+This is an alpha feature and as such, your mileage might vary with the functionality of this feature. Support will be provided on a best effort basis. Since this is an alpha feature, it should go without saying that this should never be enabled in a production environment. The feature minimally requires NSX-T 2.5 or higher.
```

Comment:
Can you remind me why it's 2.5 again?

Reply:
When the issue was first raised, I messaged @mandelsoft in the k8s slack about what version was tested and implemented with when submitting the PR.

`We are using version 2.5. Our first implementation was based on the advanced api.`